### PR TITLE
SDK-117 : Add X-Yoti-SDK header to the request and remove Fullname attribute

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: src/Yoti/YotiClient.php [hunk 1]
```diff
@@ -8,24 +8,32 @@
...
     const CONNECT_BASE_URL = 'https://www.yoti.com/connect';
 
-    // dashboard login
+    // Dashboard login
     const DASHBOARD_URL = 'https://www.yoti.com/dashboard';
 
+    // Accepted HTTP header values for X-Yoti-SDK header
+    const YOTI_ACCEPTED_SDK_HEADERS = [
+        'PHP',
+        'Wordpress',
```
Comment (vassyz) [inline src/Yoti/YotiClient.php]: Shouldn't it be `WordPress`

File: src/Yoti/YotiClient.php [hunk 2]
```diff
@@ -52,50 +60,63 @@ class YotiClient
     private $_mockRequests = false;
 
     /**
+     * @var string
+     */
+    private $_yotiSdkHeader;
```
Comment (vassyz) [inline src/Yoti/YotiClient.php]: Other SDKs are using the name `sdkIdentifier`, up to you if you want to use the same.

Comment (sidibos) [reply src/Yoti/YotiClient.php]: yes I will use the sdkIdentifier instead to keep consistency.

File: src/Yoti/YotiClient.php (hunk 1 above)
Comment (sidibos) [reply src/Yoti/YotiClient.php]: Ok I will use `WordPress`

**Pull request merged**